api/utils/keys/yubikey_test.go

```diff
@@ -44,16 +45,28 @@ func TestGetYubiKeyPrivateKey_Interactive(t *testing.T) {
 	fmt.Println("This test is interactive, tap your YubiKey when prompted.")
 
 	ctx := context.Background()
-	resetYubikey(ctx, t)
+	t.Cleanup(func() { resetYubikey(t) })
+
+	// Handle PIN prompts.
+	testPIN := "123123"
+	oldStdin := prompt.Stdin()
+	t.Cleanup(func() { prompt.SetStdin(oldStdin) })
+	prompt.SetStdin(&mockPinReader{testPIN})
```

Comment:
```suggestion
	prompt.SetStdin(prompt.NewFakeReader().AddString(testPIN))
```

and drop mockPinReader.

Reply:
This is fine, it represents the test anyway. Thanks.